packages/components/src/DataList/DataList.css

```diff
@@ -35,6 +35,33 @@
   border-bottom: var(--border-base) solid var(--color-border);
 }
 
+.selectable {
+  display: grid;
+  column-gap: var(--space-small);
+  align-items: flex-start;
+  grid-template-columns: max-content minmax(0px, auto);
```

Comment:
You sent me a message about `auto` not working well. Especially on smaller screens.

The answer to that is because, like flex, grid also adds a `min-width` of `auto` on the target element [by default](https://atlantis.getjobber.com/?path=/story/components-selections-chips-web--basic). Unfortunately, some elements that are meant to not wrap and overflow aren't working well with it. So the fix is to tell that child to have a `min-width` of 0. Or what you did here. Which cancels out that issue.

Reply:
@darryltec thanks for the explanation as to what is going on